cmake/CompileProtos.cmake

```diff
@@ -262,8 +262,23 @@ endfunction ()
 #
 function (google_cloud_cpp_load_protodeps var file)
     file(READ "${file}" contents)
-    string(REGEX REPLACE "\n" ";" contents "${contents}")
+    string(REPLACE "\n" ";" contents "${contents}")
     set(deps)
+
+    # Omit a target from deps.
+    set(targets_to_omit
```

Comment:
> I wonder if it would be cleaner to add these special cases to the google/cloud/*/CMakeLists.txt file that need them

Maybe?  At the moment I thought it better that the `CMakeLists.txt` files all looked the same (the `orgpolicy.proto` case notwithstanding), and that the exceptions were co-located (particularly when they affect multiple libraries).  But we can certainly revisit.